config/manager/kustomization.yaml

```diff
@@ -12,5 +12,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: controller
+  newName: quay.io/rabi/openstack-operator
```

Comment:
`s/rabi/openstack-k8s-operators`.  Or, instead, we can add this file to `.gitignore` as the newer versions of Operator SDK set by default during scaffolding: https://github.com/openstack-k8s-operators/openstack-baremetal-operator/blob/6df055e5341bef713e051364e7b38b34b5162bec/.gitignore#L23

Reply:
Yeah, I have it in .gitignore now, not sure how it made it to the PR earlier.